src/components/SingleTweet.tsx

```diff
@@ -0,0 +1,224 @@
+import { Dropdown } from "react-bootstrap";
+import styled from "styled-components";
+
+const Wrapper = styled.div`
+  max-width: 598px;
+  padding: 0 16px;
+  text-align: left;
+`;
+const Author = styled.div`
+  height: 48px;
+  margin-top: 12px;
+  margin-bottom: 4px;
+  display: flex;
+  align-items: center;
+  letter-spacing: -0.02em;
+`;
+const Logo = styled.img`
+  margin-right: 12px;
+  border-radius: 50%;
+`;
+const NameSection = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+`;
+const NameWrapper = styled.div``;
+const Name = styled.div`
+  font-weight: bold;
```

Comment:
font customization can be solved with bootstrap Text modifications
https://getbootstrap.com/docs/4.0/utilities/text/#font-weight-and-italics

fix here and everywhere